addons/calendar/models/calendar_event.py

```diff
@@ -132,6 +132,7 @@ def _default_stop(self):
     access_token = fields.Char('Invitation Token', store=True, copy=False, index=True)
     videocall_source = fields.Selection([('discuss', 'Discuss'), ('custom', 'Custom')], compute='_compute_videocall_source')
     videocall_channel_id = fields.Many2one('discuss.channel', 'Discuss Channel')
+    _unique_channel = models.UniqueIndex("(videocall_channel_id)")
```

Comment:
This will require an upgrade script to deduplicate or remove the existing duplicates.